### DISCUSSION TITLE
Azure Container Apps configuration

### DISCUSSION THREAD

**Discussion by christiannagel** [microsoft/aspire#2069] [General]
How is it possible to configure the minimum/maximum values for replicas, and the container allocated CPI cores and memory?
Default values are 0.5 cores and 1Gi memory, and 1-10 replicas. 

How can I change this configuration with Aspire?

**Comment by davidfowl** [comment]
Locally there's no control over memory or CPU. You can control the replicas using the WithReplicas extension method. For azure container apps, you need to use azd infra synth and tweak the yaml and bicep to control specifics in the deployed environment.

**Reply by christiannagel** [reply]
I tried `azd infra synth` to do this. 
`azd up` successfully created Container Apps for the `AddProject` and `AddRedisContainer` invocations earlier.
`azd infra synth` created bicep definitions only for the redis container, but not the projects. Is this a manual step, and can this then still be used with `azd up`?

**Reply by davidfowl** [reply]
Seems to be an issue https://github.com/Azure/azure-dev/issues/3179

**Reply by christiannagel** [reply]
Many thanks @davidfowl! Checking into this azure-dev issue.

**Reply by AlpineJBoehnen** [reply]
@davidfowl Hello, could you please confirm that there is still no way to control memory or CPU locally?

**Reply by davidfowl** [reply]
No, please open an issue.

**Reply by AlpineJBoehnen** [reply]
Thank you for confirming! https://github.com/dotnet/aspire/issues/8735
